### PR TITLE
fix a couple bugs

### PR DESCRIPTION
**Pull request by ZappoMan**
- add message mixer pings
- don't report delete entity warnings by default

### CODE REVIEW

**Comment by ZappoMan** [comment]
Fixes the following bugs:
https://app.asana.com/0/26225263936266/98971427181956
https://app.asana.com/0/26320649018758/98690983358503

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1c0727650123448d612cbee2c09dc6ba4c9aec49 available at: https://gist.github.com/f46fb01cddd4b97e967d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7289.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7289.dmg

**Comment by ZappoMan** [comment]
Test Notes:
- Run the app... show stats (with pings), click to expand stats, you should see a Message Mixer Ping
- Open console and type: `Entities.deleteEntity("11111111-1111-1111-1111-111111111111");` and notice that no warning is displayed in the log

**Comment by OmegaHeron** [comment]
Unable to test - immediate crash on open with Windows 7, bugsplats submitted.

**Comment by howard-stearns** [comment]
@ZappoMan - Re @OmegaHeron's comment, probably needs to merge from upstream master (to get https://github.com/highfidelity/hifi/pull/7293)

File: libraries/entities/src/EntityTree.h [hunk 1]
```diff
@@ -114,8 +114,8 @@ class EntityTree : public Octree {
     // use this method if you have a pointer to the entity (avoid an extra entity lookup)
     bool updateEntity(EntityItemPointer entity, const EntityItemProperties& properties, const SharedNodePointer& senderNode = SharedNodePointer(nullptr));
 
-    void deleteEntity(const EntityItemID& entityID, bool force = false, bool ignoreWarnings = false);
-    void deleteEntities(QSet<EntityItemID> entityIDs, bool force = false, bool ignoreWarnings = false);
+    void deleteEntity(const EntityItemID& entityID, bool force = false, bool ignoreWarnings = true);
+    void deleteEntities(QSet<EntityItemID> entityIDs, bool force = false, bool ignoreWarnings = true);
```
Comment (howard-stearns) [inline libraries/entities/src/EntityTree.h]: I'm not certain, but I _think_ the annoying warnings were from deleteEntities (plural), occurring on, e.g. lifetime expiration. If that's true, we could keep some measure of debuggability by only ignoreWarnings for that one. 
(Your call. Again, I'm not sure that I'm remembering correctly.)

Comment (ZappoMan) [inline libraries/entities/src/EntityTree.h]: that was my original plan... but I decided unwinding that ended up touching more code than I wanted to for this simple PR... I left this notion of warnings in, so we could go back later and add them for the cases we felt were needed.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e038c8a43c41ad6aafe22a9277a9d623212c657d available at: https://gist.github.com/f46fb01cddd4b97e967d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7289.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7289.dmg

**Comment by OmegaHeron** [comment]
Tested Windows 7

Entities.deleteEntity("11111111-1111-1111-1111-111111111111"); and notice that no warning is displayed in the log.  Confirmed.

Clicked to get expanded stats and see Messages max ping listed and it matches expected value.

QA Pass

**Pull request merged**